<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>423</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>750</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>IrGene</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item alignment="Qt::AlignLeft">
     <widget class="QTreeWidget" name="treeInfo">
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tblViewResult">
      <property name="minimumSize">
       <size>
        <width>300</width>
        <height>350</height>
       </size>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <widget class="QMenu" name="menuDataBases">
     <property name="title">
      <string>Базы Данных</string>
     </property>
    </widget>
    <addaction name="btnOpen"/>
    <addaction name="btnSave"/>
    <addaction name="btnImport"/>
    <addaction name="menuDataBases"/>
    <addaction name="separator"/>
    <addaction name="btnExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Таблицы сопряженности</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Инструменты</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>Функции группировки</string>
     </property>
    </widget>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>Построить Филогенетическое дерево</string>
     </property>
    </widget>
    <addaction name="menu_4"/>
    <addaction name="action"/>
    <addaction name="menu_5"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statBar"/>
  <action name="btnOpen">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="btnSave">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="btnExit">
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="pbtnOne">
   <property name="text">
    <string>с учетом одной аллели</string>
   </property>
  </action>
  <action name="pbtnPair">
   <property name="text">
    <string>с учетом аллельной пары</string>
   </property>
  </action>
  <action name="pbtnGaplotype">
   <property name="text">
    <string>с учетом гаплотипа</string>
   </property>
  </action>
  <action name="action_GroupGom">
   <property name="text">
    <string>по гомозиготности</string>
   </property>
  </action>
  <action name="action_max">
   <property name="text">
    <string>по max относит. риска</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Визуализация</string>
   </property>
  </action>
  <action name="btnImport">
   <property name="text">
    <string>Импорт из Excel</string>
   </property>
  </action>
  <action name="actOpenPopData">
   <property name="text">
    <string>Открыть Популяционные Данные</string>
   </property>
  </action>
  <action name="actBuidTree">
   <property name="text">
    <string>Построить Филогенетическое дерево</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Построить Филогенетическое дерево</string>
   </property>
  </action>
  <action name="actionVcxz">
   <property name="text">
    <string>vcxz</string>
   </property>
  </action>
  <action name="actionF">
   <property name="text">
    <string>f</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
